files/etc/uci-defaults/99-nethsec-mwan

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
Not all uci defaults have the the shebang: not sure if is needed.

Reply:
Just to state what the file is about, uci-defaults run on `sh` anyway